mysql-test/suite/galera/t/MDEV-32738.test

```diff
@@ -0,0 +1,73 @@
+#
+# MDEV-32738: Don't BF-abort high-priority transaction waiting on a lock in InnoDB.
```

Comment:
Is this really a BF abort, or a local transaction getting priority over high-priority transaction?